src/engraving/libmscore/chord.cpp

```diff
@@ -1007,19 +1007,21 @@ void Chord::computeUp()
         for (ChordRest* cr : _beam->elements()) {
             if (cr->isChord() && toChord(cr)->staffMove() != 0) {
                 cross = true;
-                int move = toChord(cr)->staffMove();
-                // we have to determine the first and last chord direction for the beam
-                // so that we can calculate the beam anchor points
-                if (move > 0) {
-                    _up = staffMove() != 0;
-                    firstCr->setUp(firstCr->staffMove() != 0);
-                    lastCr->setUp(lastCr->staffMove() != 0);
-                } else {
-                    _up = staffMove() == 0;
-                    firstCr->setUp(firstCr->staffMove() == 0);
-                    lastCr->setUp(lastCr->staffMove() == 0);
+                if (!_beam->userModified()) { // if the beam is user-modified _up must be decided later down
+                    int move = toChord(cr)->staffMove();
+                    // we have to determine the first and last chord direction for the beam
+                    // so that we can calculate the beam anchor points
+                    if (move > 0) {
+                        _up = staffMove() > 0;
+                        firstCr->setUp(firstCr->staffMove() > 0);
+                        lastCr->setUp(lastCr->staffMove() > 0);
+                    } else {
+                        _up = staffMove() >= 0;
+                        firstCr->setUp(firstCr->staffMove() >= 0);
+                        lastCr->setUp(lastCr->staffMove() >= 0);
+                    }
+                    break;
```

Comment:
Couldn't the `break` be moved out of the `if (!_beam->userModified()) {` block?

Reply:
I did, at some point, then I somehow lost it. Done now!